services/filemanager/miniomanager.go

```diff
@@ -116,8 +124,19 @@ func GetMinioConfig(config map[string]interface{}) *MinioConfig {
 	}
 }
 
-func (manager *MinioManager) DeleteObjects(locations []string) (err error) {
-	return
+func (manager *MinioManager) DeleteObjects(keys []string) (err error) {
+	minioClient, err := minio.New(manager.Config.EndPoint, manager.Config.AccessKeyID, manager.Config.SecretAccessKey, manager.Config.UseSSL)
```

Comment:
Should a minioClient being created every time ?
Could it be part of the struct ?

Reply:
good idea